actor/client/client.go

```diff
@@ -0,0 +1,34 @@
+package client
+
+import (
+	"context"
+	"github.com/dapr/go-sdk/actor"
+	"github.com/dapr/go-sdk/actor/config"
+	"github.com/dapr/go-sdk/client"
+)
+
+type ActorClient interface {
```

Comment:
I adding this is to ask, should we split "actor client" from grpcClient interface? or just use the old one, adding these actor api to it?

Reply:
Let's add the actors API to the existing client.

Reply:
ping @LaurenceLiZhixin

Reply:
> Let's add the actors API to the existing client.

OK